content/docs/2.7/operate/cluster.md

```diff
@@ -65,6 +65,19 @@ $env:KEDA_HTTP_DEFAULT_TIMEOUT=1000
 
 All applicable scalers will use this timeout. Setting a per-scaler timeout is currently unsupported.
 
+## HTTP connection disable keep alive
+You can override this default by setting the `DISABLE-KEEP-ALIVES` environment variable to true. For example, on Linux/Mac/Windows WSL2 operating systems, you'd use this command to set to true:
```

Comment:
```suggestion
## HTTP Keep alives

Keep alive behaviour is enabled by default for every HTTP connection, this could stack a huge amount of connections (one per scaler) in some scenarios. You can override this default setting `KEDA_HTTP_DISABLE_KEEP_ALIVE` environment variable to true. For example, on Linux/Mac/Windows WSL2 operating systems, you'd use this command to set to true:
```